### PR TITLE
feat(cart): separate cart heading component

### PR DESCRIPTION
**Pull request by aleespryker**
The separation of the cart heading and the cart entries improves the reusability of both elements and will help to add more features to the cart heading going forward. Consequently, we introduced a checkout and order heading. 

The heading components are considered new features that will official land in 2.0, but can already be used and applied automatically when the `ORYX_FEATURE_VERSION` (min. `1.2`) is used.

closes: [HRZ-3262](https://spryker.atlassian.net/browse/HRZ-3244)

[HRZ-3262]: https://spryker.atlassian.net/browse/HRZ-3262?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/domain/cart/heading/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './heading.component';
```
Comment (supproduction) [inline libs/domain/cart/heading/index.ts:1]: do we want to export styles and def?

File: libs/domain/order/heading/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from './heading.component';
```
Comment (supproduction) [inline libs/domain/order/heading/index.ts:1]: same for styles and def

File: libs/template/presets/storefront/app/1_1/experience/experience-data.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  editAddressPage,
+  homePage,
+  loginPage,
+  productPage,
+  searchPage,
+} from '../../1_0/experience/pages';
+import { cartPage, checkoutPage, orderConfirmationPage } from './pages';
+
+export const staticExperienceLatestFeature: AppFeature = {
+  providers: [
```
Comment (supproduction) [inline libs/template/presets/storefront/app/1_1/experience/experience-data.ts]: Can we just use featureVersion instead of copy paste templates?

**Review by supproduction — CHANGES_REQUESTED** [review]

File: libs/domain/order/heading/index.ts (hunk 2 above)
Comment (tobi-or-not-tobi) [reply libs/domain/order/heading/index.ts:1]: there are no styles and we don't want to export def

File: libs/domain/cart/heading/index.ts (hunk 1 above)
Comment (tobi-or-not-tobi) [reply libs/domain/cart/heading/index.ts:1]: yes to styles, no to def (as it breaks lazy loading)

File: libs/template/presets/storefront/app/1_1/experience/experience-data.ts (hunk 3 above)
Comment (tobi-or-not-tobi) [reply libs/template/presets/storefront/app/1_1/experience/experience-data.ts]: actually, big left over, as I dropped the full copy/paste templates as part of the revamp of this one ;) So we think the same direction, just forgot to drop this stuff.

File: libs/domain/checkout/heading/index.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export * from './heading.component';
+export * from './heading.def';
```
Comment (supproduction) [inline libs/domain/checkout/heading/index.ts]: Drop def here, please

**Review by supproduction — CHANGES_REQUESTED** [review]

**Review by supproduction — APPROVED** [review]

**Pull request merged**